doc/data/messages/u/unreachable-match-patterns/bad.py

```diff
@@ -0,0 +1,13 @@
+red = 0
+green = 1
+blue = 2
+
+
+color = blue
+match color:
+    case red:  # [unreachable-match-patterns]
+        print("I see red!")
+    case green:  # [unreachable-match-patterns]
+        print("Grass is green")
+    case blue:
+        print("I'm feeling the blues :(")
```

Comment:
I'm a little torn, but I guess I would expect to see this message on the second and third cases, not the first and second, since it's the second and third that are unreachable. We don't know if the fix is to alter the name capture or to remove the extra cases.

Reply:
Fair point. the mention of "unreachable-match-patterns" on the problematic line may create the impression that this line is itself unreachable.

So, we can:
1. change the name of the message (to something other than 'unreachable-match-patterns') or
2. keep the name as it currently is but move it to the lines which are unreachable.

I don't feel terribly strongly for either 1. or 2. but my feeling is I prefer to emit the message on the line(s) causing the problem, so that the root cause is immediately identifiable. I hope the message text can give enough context to be clear what is going on:
**The name capture `case a` makes the remaining patterns unreachable. Use a dotted name(for example an enum) to fix this**